financial/reports/report_xlsx_financial_defaults.py

```diff
@@ -0,0 +1,563 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 KMEE INFORMATICA LTDA
+#    Luiz Felipe do Divino <luiz.divino@kmee.com.br>
+# License AGPL-3 or later (http://www.gnu.org/licenses/agpl)
+#
+
+from __future__ import division, print_function, unicode_literals
+import html2text
```

Comment:
And https://travis-ci.org/OCA/l10n-brazil/jobs/260387688#L622 because filter branch exclude your add in requirements.txt